addons/mail/static/src/core/common/message_reaction_list.js

```diff
@@ -13,12 +14,19 @@ export class MessageReactionList extends Component {
 
     setup() {
         super.setup();
+        this.loadEmoji = loadEmoji;
         this.store = useState(useService("mail.store"));
         this.ui = useService("ui");
         this.hover = useHover(["reactionButton", "reactionList*"], {
             onHover: () => (this.preview.isOpen = true),
             onAway: () => (this.preview.isOpen = false),
         });
+        this.state = useState({ emojiLoaded: loader.loaded && true });
```

Comment:
`&& true`?

I don't get it

Reply:
It's because `loaded` is truthy and contains loaded data, the `&& true` is to enforce boolean. I could have used `Boolean(loader.loaded)`, this works the same